src/components/board/Board.jsx

```diff
@@ -1,3 +1,56 @@
-const Board = () => {};
+import Image from "next/image";
+const Board = ({ name, role, yearMajor, funFact, picture }) => {
+  return (
+    <div className="container w-60 h-90 bg-biscuits-yellow shadow-[20px_-15px_0px_rgb(190,156,106)]">
+      <div className="justify-center">
+        <Image
+          src={picture}
+          alt="picture"
+          className="aspect-square object-scale-down object-contain h-40 w-64"
+          width={100}
+          height={100}
+        />
+
+        <div className="text-wrap text-center">
+          <p
+            className={
+              "text-biscuits-green-200 font-shrikhand text-xl p-1 -translate-y-1"
+            }
+          >
```

Comment:
```suggestion
            className="text-biscuits-green-200 font-shrikhand text-xl p-1 -translate-y-1"
```

unless you are putting javascript variables in class name, keep it as a double quotations. this will make prettier format to one line and make it easier to read

Reply:
apply to all of the other class names

Reply:
there is also extra p tags nested in div tags. you can just use the div tags itself.

Reply:
make the name a little bit bigger